splinterd/man/splinterd.1.md

```diff
@@ -1,81 +1,278 @@
-% COMMAND(1) Cargill, Incorporated | Splinter Commands
-
-<!--
-Template for a man page source file in Markdown format. The file should have the
-name of the command plus any subcommands (with dashes) with an extension of
-`1.md`. For example `splinter-cert-generate.1.md` is the Markdown file that
-generates the man page for `splinter cert generate`.
--->
+% SPLINTERD(1) Cargill, Incorporated | Splinter Commands
+
 NAME
 ====
 
-**command-subcommand** — About blurb from command help
+**splinterd** — Starts the Splinter daemon
 
 SYNOPSIS
 ========
-| **command** \[**FLAGS**\] \[**OPTIONS**\]
+
+**splinterd** \[**FLAGS**\] \[**OPTIONS**\] <arguments>
 
 DESCRIPTION
 ===========
-Paragraph or two that describes what the command does, explains how it fits in
-the Splinter environment, provides big-picture information on who uses it and
-why (or when), and lists prerequisites or other considerations.
+
+This command configures and runs the Splinter daemon, `splinterd`. This daemon
+provides core Splinter functionality such as circuit management, a REST API,
+and an admin service. It can also include application-specific services.
+
+The `splinterd` command requires a new unique ID for the node. Other settings
+ are optional.
+
+This command provides the `-c` and `--config` options to specify a TOML
+configuration file containing splinterd settings, instead of using command-line
+options. (Any options on the command line will override those in the
+configuration file.) The file name must end with a `.toml` extension, such as
+`splinterd.toml`). For examples, see
+[splinter/splinterd/sample_configs](https://github.com/Cargill/splinter/tree/master/splinterd/sample_configs)
+in the `splinter` repository.
+
+The Splinter daemon can be run with raw TCP connections or TLS (`tcps`)
+connections. For TLS, the certificate authority (CA) certificates and keys
+are stored in `/etc/splinter/certs/` by default. If necessary, you can change
+the directory, specify individual file paths and names, or use insecure
+self-signed certificates in a development environment. For more information,
+see CERTIFICATE FILES, below.
 
 FLAGS
 =====
-<!--
-List of flags (in alphabetical order) that describes what each flag does
-and the default value/behavior. If appropriate, explain
-how/when it's used, with tips, gotchas, and very short examples.
--->
+
```

Comment:
Please add ` --no-tls      Turn off tls configuration`.  If this command is not provided and the cert and keys for tls are not available splinterd will error. This will limit splinterd to only running over tcp.